### ISSUE TITLE
Staging Review finding: Some links that say "opens in a new tab" open in the same tab

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#93002]
### Acceptance criteria
- Link behavior matches the content as described

_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** IVC Forms
**Product:** 10-10d CHAMPVA Application Form
**Feature:** new product

## Findings details
**VA.gov Experience Standard - issue:** [User is taken out of the primary flow without notification.](https://design.va.gov/about/experience-standards/#user-is-notified-if-an-action-will-take-them-out-of-the-primary-flow)
**VA.gov Experience Standard - category:** [Efficiency](https://design.va.gov/about/experience-standards/#category-efficiency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @erinrwhite (IA), @briandeconinck (Accessibility)

### Description

On [signer information page](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d/signer-type), all three links in the additional info component say “opens in a new tab”. They open in the same tab.

<details><summary>Link, screenshot or steps to recreate</summary>

![additional-info-links](https://github.com/user-attachments/assets/71d6ded0-a360-4fbf-b2dd-8612dec2b718)

</details>

## Recommended action

Set links to open in a new tab.

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** [Link ](https://design.va.gov/components/language-toggle)
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** [Links](https://design.va.gov/content-style-guide/links)
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by reiting** [comment]
@baltes as discussed, I went through and checked that links are opening correctly. I did not find any issues!
